README.md

```diff
@@ -43,6 +50,7 @@ want to use the more common QWERTY keybindings, you'd change to:
 ```
 vim = {
     "mode": "insert",
+    "num": "",
```

Comment:
Perhaps we should also add the `"currentSequence": ""` line here? Or better yet, to prevent from having to keep this updated, maybe we can just replace this with:
```javascript
vim = {
    ...
    "keys": {
        "move": "hjkl",
        "escapeSeq": "jk",
    }
};
```
and have the user pattern-match the "..." themselves.